proto/public/uber/cadence/api/v1/domain.proto

```diff
@@ -30,33 +30,29 @@ option java_package = "com.uber.cadence.api.v1";
 import "google/protobuf/duration.proto";
 import "google/protobuf/timestamp.proto";
 
-message DomainInfo {
+message Domain {
   string id = 1;
   string name = 2;
   DomainStatus status = 3;
   string description = 4;
   string owner_email = 5;
   map<string, string> data = 6;
-}
-
-message DomainConfiguration {
-  google.protobuf.Duration workflow_execution_retention_period = 1;
-  BadBinaries bad_binaries = 2;
-  ArchivalStatus history_archival_status = 3;
-  string history_archival_uri = 4;
-  ArchivalStatus visibility_archival_status = 5;
-  string visibility_archival_uri = 6;
+  google.protobuf.Duration workflow_execution_retention_period = 7;
+  BadBinaries bad_binaries = 8;
+  ArchivalStatus history_archival_status = 9;
+  string history_archival_uri = 10;
+  ArchivalStatus visibility_archival_status = 11;
+  string visibility_archival_uri = 12;
+  string active_cluster_name = 13;
+  repeated ClusterReplicationConfiguration clusters = 14;
+  int64 failover_version = 15;
```

Comment:
This field looks like an internal implementation detail. Shall we expose this info to users?

Reply:
This is currently used within `checkOngoingFailover`. But in general I agree, this looks like an implementation detail.